src/main/java/edu/kit/datamanager/metastore2/web/impl/SchemaRegistryControllerUIImpl.java

```diff
@@ -165,7 +165,7 @@ private JSONObject getJsonObject(String path) {
         TabulatorItems[] items = null;
         Resource resource = new ClassPathResource(path);
         try {
-            items = mapper.readValue(Files.newBufferedReader(Paths.get(resource.getURI()), StandardCharsets.UTF_8), TabulatorItems[].class);
+            items = mapper.readValue(new InputStreamReader(resource.getInputStream(), StandardCharsets.UTF_8), TabulatorItems[].class);
```

Comment:
If file is located outside of the jar-file original code should work too!?

Reply:
i'm not quite sure what u mean.
The version with the InputStreamReader works locally and in docker (as far as i tested), the current version only locally. I think this could be a c&p mistake since line 150 does use InputStreamReader already for the same resources.